dockerfiles/cross-build/Dockerfile.centos-8

```diff
@@ -7,11 +7,9 @@ ARG CREATE_USER="build"
 ARG USER_OPTIONS=""
 ENV PATH /go/bin:/usr/local/go/bin:$PATH
 
-RUN yum install -y /usr/bin/bash /usr/bin/git /usr/bin/make \
-    /usr/bin/grep /usr/bin/sed /usr/bin/tr /usr/bin/find \
-    /usr/bin/sort /usr/bin/uniq \
-    /usr/bin/wget /usr/bin/tar /usr/bin/bzip2 \
-    kernel-devel rpm-build clang gcc
+RUN yum install -y rpm-build \
```

Comment:
in centos 8 yum is obsoleted in favour of dnf.

Reply:
Ok, I can fix that one and push once more.